underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -80,15 +80,24 @@ public SwiftUnderFileSystem(AlluxioURI uri, Configuration configuration) {
     String containerName = uri.getHost();
     LOG.debug("Constructor init: {}", containerName);
     AccountConfig config = new AccountConfig();
-    config.setUsername(configuration.get(Constants.SWIFT_USER_KEY));
-    config.setTenantName(configuration.get(Constants.SWIFT_TENANT_KEY));
-    config.setPassword(configuration.get(Constants.SWIFT_API_KEY));
+    if (configuration.containsKey(Constants.SWIFT_API_KEY)) {
```

Comment:
Would this mean the user should only provide either apiKey or passwordKey, and if both provided, one of them will be ignored ? If so, shall we prioritize password key over apiKey?